micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,110 @@
+  // Example: If messages stored module as: DIGIT_MDMS_RAINMAKER-PGR.SERVICEDEFS
+  // Convert rawSchemaCode: "RAINMAKER-PGR.ServiceDefs" -> "RAINMAKER-PGR.SERVICEDEFS"
+  const localizationModule = `DIGIT_MDMS_${rawSchemaCode}`.toUpperCase();
+
+  const createLocalizationCode = (fieldName, fieldValue) => {
+    const upperFieldName = fieldName.toUpperCase();
+    const transformedValue = (fieldValue || "").replace(/\s+/g, "").toUpperCase();
+    return `${rawSchemaCode}_${upperFieldName}_${transformedValue}`.toUpperCase();
+  };
+
+  let localizationCodes = [];
+  if (data && data.data && localisableFields.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)